lib/msal-core/src/Storage.ts

```diff
@@ -179,15 +179,18 @@ export class Storage {// Singleton
     }
 
     /**
-     * Create acquireTokenUserKey to cache user object
+     * Create acquireTokenAccountKey to cache account object
+     * @param accountId
+     * @param state
      */
-    static generateAcquireTokenUserKey(userId: any, state: string): string {
+    static generateAcquireTokenAccountKey(accountId: any, state: string): string {
```

Comment:
should this have a type

Reply:
no. It is not supposed to as it is a key off with no structure reference from what I understand. I can check again.